viewer_server/client_db_access_handler.py

```diff
@@ -487,20 +491,21 @@ def __update_suppress_storage_data(self,
         source_file = session.query(File).get(report.file_id)
         source_file_path, source_file_name = ntpath.split(source_file.filepath)
 
-        LOG.debug('Updating suppress data for: ' + str(report_id) + ' bug id ' +
-                  bug_id_hash + ' file name ' + source_file_name +
+        LOG.debug('Updating suppress data for: ' + str(report_id) +
+                  ' bug id ' + bug_id_hash +
+                  ' file name ' + source_file_name +
                   ' suppressing ' + str(suppress))
 
         # Check if it is already suppressed for any run ids.
         suppressed = session.query(SuppressBug) \
-            .filter(or_( \
-            and_(SuppressBug.hash == bug_id_hash,
-                 SuppressBug.file_name == source_file_name,
-                 SuppressBug.run_id.in_(run_ids)),
-            and_(SuppressBug.hash == bug_id_hash,
-                 SuppressBug.file_name == '',
-                 SuppressBug.run_id.in_(run_ids))
-        )) \
+            .filter(or_(
+                and_(SuppressBug.hash == bug_id_hash,
+                     SuppressBug.file_name == source_file_name,
+                     SuppressBug.run_id.in_(run_ids)),
+                and_(SuppressBug.hash == bug_id_hash,
```

Comment:
Do we plan to support the case when the suppress file name is empty? If we break the schema in 6.0 anyways, maybe we should delete that part of the code. (We could open a ticket for that.)